pkg/cache/ttl.go

```diff
@@ -100,23 +100,38 @@ func (c *TTL) GetKeys() []uint64 {
 }
 
 // Remove eliminates an item from cache.
-func (c *TTL) Remove(key uint64) {
+func (c *ttlCache) remove(key interface{}) {
 	c.Lock()
 	defer c.Unlock()
 
 	delete(c.items, key)
 }
 
+// pop one key/value that is not expired. If boolean is false, it means didnt find the valid one.
+func (c *ttlCache) pop() (interface{}, interface{}, bool) {
```

Comment:
How about adding some tests for this method?

Reply:
updated.